File: spec/code_samples/Node_unirest/list@check_email@{email}/get.js
var unirest = require("unirest");

var req = unirest("GET", "http://example.com/api/v1/list/check_email/%7Bemail%7D");

req.query({
  "username": "SOME_STRING_VALUE"
});

req.headers({
  "authorization": "Bearer REPLACE_BEARER_TOKEN",
  "content-type": "application/json"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Go_native/list@check_email@{email}/get.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://example.com/api/v1/list/check_email/%7Bemail%7D?username=SOME_STRING_VALUE"

	req, _ := http.NewRequest("GET", url, nil)

	req.Header.Add("content-type", "application/json")
	req.Header.Add("authorization", "Bearer REPLACE_BEARER_TOKEN")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}